listenbrainz/db/dump_manager.py

```diff
@@ -260,13 +335,32 @@ def _cleanup_dumps(location):
     else:
         remove_dumps(location, full_dumps, NUMBER_OF_FULL_DUMPS_TO_KEEP)
 
-    incremental_dump_re = re.compile('listenbrainz-dump-[0-9]*-[0-9]*-[0-9]*-incremental')
-    dump_files = [x for x in os.listdir(location) if incremental_dump_re.match(x)]
-    incremental_dumps = [x for x in sorted(dump_files, key=get_dump_id, reverse=True)]
+    # Clean up incremental dumps
+    incremental_dump_re = re.compile(
+        'listenbrainz-dump-[0-9]*-[0-9]*-[0-9]*-incremental')
+    dump_files = [x for x in os.listdir(
```

Comment:
could do `os.listdir` once at the beginning of this cleanup, but probably not computationally expensive given the number of items in this directory

Reply:
Its trivial and everything is going to be in disk cache anyways.